block-node/server/docs/design/block-content-proof.md

```diff
@@ -0,0 +1,208 @@
+# Design Document Template
+
+## Table of Contents
+
+1. [Purpose](#purpose)
+2. [Goals](#goals)
+3. [Terms](#terms)
+4. [Entities](#entities)
+5. [Design](#design)
+6. [Sequence Diagram](#sequence-diagram)
+7. [Configuration](#configuration)
+8. [Metrics](#metrics)
+9. [Exceptions](#exceptions)
+10. [Acceptance Tests](#acceptance-tests)
+
+## Purpose
+
+The purpose of this document is to define the design of the BlockContentProof feature.
```

Comment:
nit: we should standardize but initially i thought it was block contents.
Here I see Block Content.
I block has contents but I'm guessing the singular form is because I since content items (BlockItem) is under consideration.